js/packages.js

```diff
@@ -1,104 +0,0 @@
-const packages = [
-    {
-      name: "small square",
-      x: 0,
-      y: 0,
-      w:50,
-      h:50
-    },
-    // {
-    //   name: "big square",
-    //   x: 0,
-    //   y: 0,
-    //   w:200,
-    //   h:200
-    // },
-    {
-      name: "long rec horizontal",
-      x: 0,
-      y: 0,
-      w:200,
-      h:50
-    },
-    // {
-    //   name: "long rec vertical",
-    //   x: 0,
-    //   y: 0,
-    //   w:50,
-    //   h:200
-    // },
-    {
-      name: "short rect horizontal",
-      x: 0,
-      y: 0,
-      w:100,
-      h:50
-    }
-    // {
-    //   name: "short rec vertical",
-    //   x: 0,
-    //   y: 0,
-    //   w: 50,
-    //   h: 100
-    // },
-    // {
-    //   name: "L shape",
-    //   x: 0,
-    //   y: 0,
-    //   w: 50,
-    //   h: 100,
-    //   secX:  + 50,
-    //   secY:  + 50,
-    //   secW: 50,
-    //   secH: 50
-    // },
-    // {
-    //   name: "L shape reverse",
-    //   x: 0,
-    //   y: 0,
-    //   w: 50,
-    //   h: 100,
-    //   secX:  - 50,
-    //   secY:  + 50,
-    //   secW: 50,
-    //   secH: 50
-    // },
-    
-    
-  ]
-
-
-
-      //small square
-    // fill(500);
-    // rect(x, y, 50 ,50);
-
-    //big square
-    // fill(500);
-    // rect(x, y, 200 ,200);
-
-    // long rec horizontal
-    // fill(500);
-    // rect(x ,y, 200, 50)
-
-    //long rec vertical
-    // fill(500);
-    // rect(x, y, 50, 200)
-
-    //short rect horizontal
-    // fill(500);
-    // rect(x ,y, 100, 50)
-
-    //short rec vertical
-    // fill(500);
-    // rect(x, y, 50, 100)
-
-    // L shape
-    // fill(500);
-    // rect(x, y, 50, 100);
-    // rect(x + 50, y + 50, 50, 50);
-
-    // L shape flip
-    // fill(500);
-    // rect(x, 0, 50, 100);
-    // rect(x - 50, y + 50, 50, 50);
```

Comment:
While this works, a better solution would be to create a Package class so you can create new types of packages of different sizes easier and during runtime instead of having them static